Traceback (most recent call last):
  File "/Users/sumeetkumar/anaconda3/envs/jupyter_nb/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/sumeetkumar/anaconda3/envs/jupyter_nb/lib/python3.8/site-packages/nbclient/client.py", line 1117, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/sumeetkumar/anaconda3/envs/jupyter_nb/lib/python3.8/site-packages/nbclient/util.py", line 78, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/sumeetkumar/anaconda3/envs/jupyter_nb/lib/python3.8/site-packages/nbclient/util.py", line 57, in just_run
    return loop.run_until_complete(coro)
  File "/Users/sumeetkumar/anaconda3/envs/jupyter_nb/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/Users/sumeetkumar/anaconda3/envs/jupyter_nb/lib/python3.8/site-packages/nbclient/client.py", line 558, in async_execute
    await self.async_execute_cell(
  File "/Users/sumeetkumar/anaconda3/envs/jupyter_nb/lib/python3.8/site-packages/nbclient/client.py", line 862, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/sumeetkumar/anaconda3/envs/jupyter_nb/lib/python3.8/site-packages/nbclient/client.py", line 765, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from IPython.display import Video

Video("../assets/ElasticBlade_4.mp4",width=350,html_attributes="loop autoplay")
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
[0;32m/var/folders/27/8tl2p5616cj22kvkhsy0gjcw0000gn/T/ipykernel_28173/623332556.py[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m [0;32mfrom[0m [0mIPython[0m[0;34m.[0m[0mdisplay[0m [0;32mimport[0m [0mVideo[0m[0;34m[0m[0;34m[0m[0m
[1;32m      2[0m [0;34m[0m[0m
[0;32m----> 3[0;31m [0mVideo[0m[0;34m([0m[0;34m"../assets/ElasticBlade_4.mp4"[0m[0;34m,[0m[0mwidth[0m[0;34m=[0m[0;36m350[0m[0;34m,[0m[0mhtml_attributes[0m[0;34m=[0m[0;34m"loop autoplay"[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m~/anaconda3/envs/jupyter_nb/lib/python3.8/site-packages/IPython/core/display.py[0m in [0;36m__init__[0;34m(self, data, url, filename, embed, mimetype, width, height, html_attributes)[0m
[1;32m   1408[0m                 [0;34m"Consider passing Video(url='...')"[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[1;32m   1409[0m             ])
[0;32m-> 1410[0;31m             [0;32mraise[0m [0mValueError[0m[0;34m([0m[0mmsg[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   1411[0m [0;34m[0m[0m
[1;32m   1412[0m         [0mself[0m[0;34m.[0m[0mmimetype[0m [0;34m=[0m [0mmimetype[0m[0;34m[0m[0;34m[0m[0m

[0;31mValueError[0m: To embed videos, you must pass embed=True (this may make your notebook files huge)
Consider passing Video(url='...')
ValueError: To embed videos, you must pass embed=True (this may make your notebook files huge)
Consider passing Video(url='...')

